execution-engine/contract/src/contract_api/runtime.rs

```diff
@@ -67,9 +74,12 @@ pub fn call_contract<A: ArgsParser, T: CLTyped + FromBytes>(c_ptr: ContractRef,
     bytesrepr::deserialize(dest).unwrap_or_revert()
 }
 
-/// Takes the name of a function to store and a contract URef, and overwrites the value under
-/// that URef with a new Contract instance containing the original contract's named_keys, the
-/// current protocol version, and the newly created bytes of the stored function.
+/// Takes the name of an `extern "C"` function to store as a contract under the given [`URef`] which
```

Comment:
Perhaps something like

```suggestion
/// Takes the name of a (non-mangled) `extern "C"` function to store as a contract under the given [`URef`] which
```